.gitattributes

```diff
@@ -49,7 +49,7 @@
 # is only available from the command line. Turn it on by uncommenting the 
 # entries below.
 ###############################################################################
-*.md   diff=astextplain
+#*.md   diff=astextplain
```

Comment:
I assume you also have troubles seeing the `git diff` for .md files when this line is uncommented.

Reply:
I was getting errors running `git diff` on the README before commenting out this line. (Other files were fine). I couldn't find any docs on what `astextplain` did, and commenting out this line fixed the issue.

<img width="896" alt="image" src="https://user-images.githubusercontent.com/993909/195651982-45bc2d05-610e-491d-87a0-af8016bdd426.png">